lpa-submissions-e2e-tests/cypress/integration/lpa-submission-tasklist/lpa-submission-tasklist.js

```diff
@@ -0,0 +1,9 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Given('the lpa tasks are presented',()=>{
+    cy.visitTaskListQuestionnaireSubmissionPage();
+    
+});
+Given('And the task {string} is available for selection',()=>{
+
+});
```

Comment:
Add newline at end - use [editorconfig](https://editorconfig.org/) to do this for you